lib/ex_huobi/auth.ex

```diff
@@ -0,0 +1,143 @@
+defmodule Huobi.Test do
```

Comment:
`Huobi.Test` should be `Huobi.Auth`

Reply:
@kimquy This file should be removed. I use it for testing. I will delete it now.